canopen/pdo/__init__.py

```diff
@@ -0,0 +1,62 @@
+from .base import PdoBase, Maps, Map, Variable
+
+import logging
+import collections
+import itertools
+import canopen
+
+logger = logging.getLogger(__name__)
+
+
+class PDO(PdoBase):
+    """PDO Class for backwards compatibility
+    :param rpdo: RPDO object holding the Receive PDO mappings
+    :param tpdo: TPDO object holding the Transmit PDO mappings
+    """
+
+    def __init__(self, node, rpdo, tpdo):
+        super(PDO, self).__init__(node)
+        self.rx = rpdo.map
+        self.tx = tpdo.map
+        self.map = dict(itertools.chain(self.rx.iteritems(), self.tx.iteritems()))
```

Comment:
`iteritems()` is not available anymore on Python 3. Try with just `items()`.

Reply:
Of course you are right, thanks. I have solved the issue and by doing so added another way to access the PDO! (added some tests to validate this)

Now we can access the PDOs in three forms:

1. Traditional way node.pdo.tx[1]
2. Simplified way node.tpdo[1]
3. New way node.pdo[0x1A00]

I think now everything works as expected, what do you say?

Reply:
That should work. Yes I don't think I have anything else to comment on. Not sure why Travis isn't triggering and why there are still conflicts with lss.py. Do you want to merge it now? Would you mind if I squashed your commits?

Reply:
Do what you think is best @christiansandberg, i'm glad I can help.
The conflicts with the file ``canopen/lss.py`` I think is due to a mistake I did when auto formatting the files with my IDE, then the codacy bot was complaining about *trailing white spaces* so I removed them causing the conflict, you can (I think) ignore it. 

I'm finishing a new ds402 profile implementation, based on what was already available but with more functionality.

Then I might release a ROS wrapper around this library our in a separated project with this one as submodule. 

Thanks for all the help @christiansandberg.